erpnext/projects/report/delayed_tasks_summary/delayed_tasks_summary.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+/* eslint-disable */
+
+frappe.query_reports["Delayed Tasks Summary"] = {
+	"filters": [
+		{
+			"fieldname": "from_date",
+			"label": __("From Date"),
+			"fieldtype": "Date",
+			"default": frappe.datetime.add_months(frappe.datetime.get_today(), -1),
+			"reqd": 1
+		},
+		{
+			"fieldname": "to_date",
+			"label": __("To Date"),
+			"fieldtype": "Date",
+			"default": frappe.datetime.get_today(),
+			"reqd": 1
+		},
+		{
+			"fieldname": "priority",
+			"label": __("Priority"),
+			"fieldtype": "Select",
+			"options": ["Low", "Medium", "High", "Urgent"],
+			"default": "Low",
+			"reqd": 1
+		},
+		{
+			"fieldname": "status",
+			"label": __("Status"),
+			"fieldtype": "Select",
+			"options": ["Open", "Working","Pending Review","Overdue","Completed"],
+			"default": "Overdue"
```

Comment:
Add a blank option for both Status and Priority filters as they are optional filters.

![image](https://user-images.githubusercontent.com/24353136/114676566-0c43fe00-9d27-11eb-8c42-5d68cc8185a2.png)